src/tables.rs

```diff
@@ -1,3 +1,4 @@
 //! ttx implementations for font tables
 
 pub mod head;
+pub mod maxp; // What does this do?
```

Comment:
This declares the module (see the [modules cheatsheet](http://doc.rust-lang.org/1.64.0/book/ch07-02-defining-modules-to-control-scope-and-privacy.html#modules-cheat-sheet)); without this the file `tables/maxp.rs` would not be included in the source tree.

Reply:
Ok thanks.